websockify/websocketproxy.py

```diff
@@ -193,8 +192,9 @@ def do_proxy(self, target):
                     self.send_ping()
 
             if tqueue: wlist.append(target)
-            if cqueue or c_pend: wlist.append(self.request)
+            if cqueue: wlist.append(self.request)
             try:
+                # blocks until there is data from one side
```

Comment:
Or it's possible to send more data.

I'm not sure if this is very useful. An understanding of `select()` is probably required anyway before touching this code.